src/binaryen-c.h

```diff
@@ -604,7 +604,7 @@ typedef void* BinaryenExportRef;
 WASM_DEPRECATED BinaryenExportRef BinaryenAddExport(BinaryenModuleRef module, const char* internalName, const char* externalName);
 BinaryenExportRef BinaryenAddFunctionExport(BinaryenModuleRef module, const char* internalName, const char* externalName);
 BinaryenExportRef BinaryenAddTableExport(BinaryenModuleRef module, const char* internalName, const char* externalName);
-BinaryenExportRef BinaryenAddMemoryExport(BinaryenModuleRef module, const char* internalName, const char* externalName);
+BinaryenExportRef BinaryenImportExport(BinaryenModuleRef module, const char* internalName, const char* externalName);
```

Comment:
What did happen here?

Reply:
I guess it's from unified imports and non-imports PR. @kripken ?

Reply:
Or did i accidentally renamed it!! 🤔

Reply:
Ha, It's my mistake. I guess I need more sleep 🤪